services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -846,14 +843,15 @@ async def _clean_expired_uploads(self) -> None:
         )
 
         # try first to upload these from S3, they might have finished and the client forgot to tell us (conservative)
+        # and concurrency set to 1 to be low resource intensive in any case
         updated_fmds = await logged_gather(
             *(
                 self._update_database_from_storage_no_connection(fmd)
                 for fmd in list_of_expired_uploads
             ),
             reraise=False,
             log=_logger,
-            max_concurrency=MAX_CONCURRENT_DB_TASKS,
+            max_concurrency=1,
```

Comment:
minor: this could be a constant

Reply:
indeed, this could also be removed from a gather call